packages/manager/modules/pci/src/projects/project/billing/history/legacy/history-legacy.constants.js

```diff
@@ -0,0 +1,5 @@
+export const SUBSIDIARY_WITH_POST_PAID_USAGE_BILLING = ['UK', 'ES', 'PT', 'PL'];
+
+export default {
+  SUBSIDIARY_WITH_POST_PAID_USAGE_BILLING,
+};
```

Comment:
**nitpick:** Use feature-flipping to define the rules.

Reply:
Here is the feature flipping PR: 
https://stash.ovh.net/projects/I2APIENDPOINTS/repos/feature-availability/pull-requests/522/overview

Reply:
You can remove this if you've feature-flipping.